packages/suite/src/views/wallet/coinmarket/common/CoinmarketPaymentType.tsx

```diff
@@ -12,7 +12,13 @@ interface CoinmarketPaymentTypeProps {
 
 export const CoinmarketPaymentType = ({ method, methodName }: CoinmarketPaymentTypeProps) => (
     <Row gap={spacings.xs}>
-        {method && <img width="24px" src={invityAPI.getPaymentMethodUrl(method)} alt="" />}
+        {method && (
+            <Image
+                imageSrc={invityAPI.getPaymentMethodUrl(method)}
```

Comment:
Would it be hard to introduce something that it would also look good in dark theme?

Reply:
Venmo and credit card is almost invisible.

Reply:
Added a white box.